src/git.ts

```diff
@@ -124,4 +125,62 @@ export class Git {
         const url = this.remoteURL(remoteName);
         return basename(url, ".git");
     }
+

```

Comment:
I don't want to manipulate the git repository.

But consider providing a separate file

```typescript
import { GitAgent } from "@nice-labs/git-rev";
const agent = GitAgent(process.cwd());
agent.commit(...);
agent.createBranch(...);
```

Reply:
```typescript
// src/git-agent.ts
export class GitAgent extends Git {
    // ... your manipulate the git repository logic ...
}

// before: update index.ts expose GitAgent
```